src/runtime/analytics-service.js

```diff
@@ -150,6 +158,14 @@ export class AnalyticsService {
     }
   }
 
+  /**
```

Comment:
It's simpler for tests to just override the `getTimestamp_` property

Reply:
Done. Old habbits from C++ :)